eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelData.scala

```diff
@@ -481,7 +481,7 @@ case class LocalParams(nodeId: PublicKey,
 case class RemoteParams(nodeId: PublicKey,
                         dustLimit: Satoshi,
                         maxHtlcValueInFlightMsat: UInt64, // this is not MilliSatoshi because it can exceed the total amount of MilliSatoshi
-                        channelReserve: Satoshi,
+                        requestedChannelReserve: Satoshi,
```

Comment:
Same as above.

Reply:
Done in https://github.com/ACINQ/eclair/pull/2237/commits/0bcf7a089b35765d8425f0ad7987b6a237e241fc